src/datasets/arrow_dataset.py

```diff
@@ -1091,6 +1092,45 @@ def from_text(
             path_or_paths, split=split, features=features, cache_dir=cache_dir, keep_in_memory=keep_in_memory, **kwargs
         ).read()
 
+    @staticmethod
+    def from_sql(
+        path_or_paths: Union[PathLike, List[PathLike]],
```

Comment:
Should this be renamed to `conn` to match pandas or stay as `path_or_paths` to match other `from_*` function?

Reply:
`con` makes the most sense to be aligned with Pandas/Dask